### PR TITLE
extend tor for mac instructions

### PR DESCRIPTION
**Pull request by BitcoinMechanic**

### CODE REVIEW

File: site/source/user-manual/connecting/connecting-tor/tor-os/tor-mac.rst [hunk 1]
```diff
@@ -57,3 +52,60 @@ Install Tor
...
+Enable Tor Systemwide
+----------------------
+
+#. Now enable apache service:
+
+    .. code-block::
+
+        sudo launchctl load -w /System/Library/LaunchDaemons/org.apache.httpd.plist
+
+#. Enable proxy autoconfig file:
```
Comment (kn0wmad) [inline site/source/user-manual/connecting/connecting-tor/tor-os/tor-mac.rst:65]: This should probably say something like "Download the Start9 standard proxy config file, or use your own"

File: site/source/user-manual/connecting/connecting-tor/tor-os/tor-mac.rst [hunk 2]
```diff
@@ -57,3 +52,60 @@ Install Tor
...
+        :width: 40%
+        :alt: System Preferences
+
+#. Click on Network:
+
+    .. figure:: /_static/images/tor/network.png
+        :width: 80%
+        :alt: Select Network
+
+#. If using WiFi, select WiFi on the left panel and click "Advanced":
```
Comment (kn0wmad) [inline site/source/user-manual/connecting/connecting-tor/tor-os/tor-mac.rst:83]: Maybe reword this to include ethernet, instead of as a note below?  Or is it entirely impossible to use ethernet on a mac these days?

Comment (BitcoinMechanic) [reply site/source/user-manual/connecting/connecting-tor/tor-os/tor-mac.rst:83]: On the laptops it is, on the Mac Pro and the iMac it isn't. As far as adapters go they all have different names so I can't give a helpful picture here given that it would only be relevant for those with the same adapter.

File: site/source/user-manual/connecting/connecting-tor/tor-os/tor-mac.rst (hunk 1 above)
Comment (BitcoinMechanic) [reply site/source/user-manual/connecting/connecting-tor/tor-os/tor-mac.rst:65]: Resolved

File: site/source/user-manual/connecting/connecting-tor/tor-os/tor-mac.rst [hunk 3]
```diff
@@ -57,3 +52,60 @@ Install Tor
...
+        :width: 80%
+        :alt: Select Automatic proxy config and enter URL
+
+#. Finally, click "Apply"
+
+    .. figure:: /_static/images/tor/applyproxy.png
+        :width: 80%
+        :alt: Apply proxy
+
+Done! You have now enabled system wide Tor potential.
```
Comment (kn0wmad) [inline site/source/user-manual/connecting/connecting-tor/tor-os/tor-mac.rst:109]: Your Mac will now handle all Tor traffic via this proxy, and there is no need to configure individual apps to use Tor

**Review by kn0wmad — CHANGES_REQUESTED** [review]
See comments, ty

File: site/source/user-manual/connecting/connecting-tor/tor-os/tor-mac.rst (hunk 3 above)
Comment (BitcoinMechanic) [reply site/source/user-manual/connecting/connecting-tor/tor-os/tor-mac.rst:109]: I don't believe so - FF does need additional configuration - @k0gen please clarify

File: site/source/user-manual/connecting/connecting-tor/tor-os/tor-mac.rst (hunk 2 above)
Comment (kn0wmad) [reply site/source/user-manual/connecting/connecting-tor/tor-os/tor-mac.rst:83]: I don't think a picture is necessary, just mention ethernet is an option, but that this example uses wifi

**Review by kn0wmad — APPROVED** [review]
